editor-components/modal/notificationmodal2/notificationmodal2.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import { BaseModal } from "../../EditorComponent";
+import styles from "./notificationmodal2.module.scss";
+import { ComposerIcon } from "../../../composer-base-components/icon/icon";
+
+class NotificationModal2 extends BaseModal {
+  constructor(props?: any) {
+    super(props, styles);
+
+    this.addProp({
+      type: "icon",
+      key: "icon",
+      displayer: "Icon",
+      value: "PiBellRingingThin",
+    });
+
+    this.addProp({
+      type: "string",
+      key: "notificationTitle",
+      displayer: "Title",
+      value: "Notifications",
+    });
+
+    this.addProp({
+      type: "string",
+      key: "displayerMessage",
+      displayer: "Description",
+      value: "Receive an email every time a new lesson is added to the website.",
+    });
+
+    this.addProp({
+      type: "string",
+      key: "email",
+      displayer: "E-mail",
+      value: "Your e-mail",
+    });
+
+    this.addProp({
+      type: "string",
+      key: "button",
+      displayer: "Button",
+      value:"Subscribe",
+    });
+  }
+
+  getName(): string {
+    return "Notification Modal 2";
+  }
+
+  render() {
+    const icon = this.getPropValue("icon");
+    const buttonval = this.getPropValue("button", {as_string: true});
+    const emailPlaceholder = this.getPropValue("email", {as_string: true});
+    const description = this.getPropValue("displayerMessage", {as_string: true});
+
+    return (
+      <div className={this.decorateCSS("container")}>
+        <div className={this.decorateCSS("max-content")}>
+          <div className={this.decorateCSS("content")}>
+            {icon && <div className={this.decorateCSS("icon")}>
+              <ComposerIcon
+                propsIcon={{ className: this.decorateCSS("Icon")}}
+                name={this.getPropValue("icon")}
+              />
+            </div>}
+            <h2 className={this.decorateCSS("notification-title")}>
+              {this.getPropValue("notificationTitle")}
+            </h2>
+            {description && (
+              <p className={this.decorateCSS("displayer-message")}>
+              {description}
+              </p>
+            )}
+            {emailPlaceholder && (
+              <div className={this.decorateCSS("send-mail")}>
+                <div className={this.decorateCSS("input-group")}>
+                  <input
+                    placeholder={emailPlaceholder}
+                    type="text"
+                    className={this.decorateCSS("input")}
+                  />
+                   {buttonval && (
+                      <button className={this.decorateCSS("button")}>
+                        {buttonval}
```

Comment:
this should be like:
```suggestion
                        {this.getPropValue("button")}
```